models/datasets/generic/ds__imaging_requests.sql

```diff
@@ -0,0 +1,94 @@
+with results as (
+    select
+        imaging_request_id,
+        min(datetime) as completed_datetime
+    from {{ ref('imaging_results') }}
+    group by imaging_request_id
+)
+
+select
+    p.id as patient_id,
+    p.display_id,
+    p.first_name,
+    p.last_name,
+    p.date_of_birth,
+    date_part('year', age(ir.datetime::date, p.date_of_birth::date)) as age,
+    initcap(p.sex::text) as sex,
+    v.id as village_id,
+    v.name as village,
+    f.id as facility_id,
+    f.name as facility,
+    d.id as department_id,
+    d.name as department,
+    lg.id as location_group_id,
+    lg.name as location_group,
+    ir.display_id as request_id,
+    ir.datetime as requested_datetime,
+    su.id as supervising_clinician_id,
+    su.display_name as supervising_clinician,
+    ru.id as requesting_clinician_id,
+    ru.display_name as requesting_clinician,
+    case
+        when ir.priority = 'routine' then 'Routine'
+        when ir.priority = 'urgent' then 'Urgent'
+        when ir.priority = 'asap' then 'ASAP'
+        when ir.priority = 'stat' then 'STAT'
+    end as priority,
```

Comment:
`today` is used in Palau. Priority appears to be a deployment config that can't be found in the database.
```suggestion
    case
        when ir.priority = 'routine' then 'Routine'
        when ir.priority = 'urgent' then 'Urgent'
        when ir.priority = 'asap' then 'ASAP'
        when ir.priority = 'stat' then 'STAT'
        when ir.priority = 'today' then 'Today'
        else ir.priority
    end as priority,
```